README.md

```diff
@@ -77,8 +77,8 @@ and DLT-META [documentation](https://databrickslabs.github.io/dlt-meta/)
 2. Create ```silver_transformations.json``` and save to s3/adls/dbfs e.g [Silver transformation file](https://github.com/databrickslabs/dlt-meta/blob/main/examples/silver_transformations.json)
 3. Create data quality rules json and store to s3/adls/dbfs e.g [Data Quality Rules](https://github.com/databrickslabs/dlt-meta/tree/main/examples/dqe/customers/bronze_data_quality_expectations.json)
 
-## 2. Onboarding job
-
+## 2. Onboarding
```

Comment:
should we add a link to getting started guide for **Steps** section? This is duplicating the content and it will reduce the overall README content.

Reply:
@ganeshchand  Removed these steps from read me and pointed to Getting started.